apps/website/src/components/FeedbackLink.astro

```diff
@@ -0,0 +1,39 @@
+---
+import ChatIcon from '~/icons/ChatIcon.astro';
+
+type Props = { page: string } & astroHTML.JSX.AnchorHTMLAttributes;
+let { page, ...rest } = Astro.props;
+
+let fileName = 'index.mdx'; // fallback value if nothing in the url
+
+if (page) {
+  fileName = `${page}`;
+}
```

Comment:
i don't think this adds anything. can use `page` directly in the feedback url

Reply:
You are correct, removed.